tests/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "edv-client-tests",
+  "version": "0.0.1",
+  "description": "Tests for the edv-client.",
+  "main": ".eslintrc.js",
+  "dependencies": {},
+  "devDependencies": {
+    "crypto-ld": "^3.7.0",
+    "did-context": "^2.0.0",
+    "did-method-key": "^0.4.0",
+    "jsonld-signatures": "^4.6.0",
+    "ocapld": "^1.6.1"
+  },
+  "scripts": {
+    "test": "mocha"
```

Comment:
not working, as mocha is not a dependency of this package

Reply:
I went ahead and moved all the test depdencies to the devDependencies in the main project: https://github.com/digitalbazaar/edv-client/pull/26/commits/c38b30db6e5404b4b7ff065205b9a6a982189cc5